src/circuit/gadget/ecc/chip.rs

```diff
@@ -151,25 +147,58 @@ impl EccChip {
         Self { config }
     }
 
+    /// # Side effects
+    ///
+    /// All columns in `advices` and `constants` will be equality-enabled.
     #[allow(non_snake_case)]
     pub fn configure(
         meta: &mut ConstraintSystem<pallas::Base>,
         advices: [Column<Advice>; 10],
         lookup_table: Column<Fixed>,
         // TODO: Replace with public inputs API
         constants: [Column<Fixed>; 2],
-        perm: Permutation,
     ) -> <Self as Chip<pallas::Base>>::Config {
-        let lookup_config = LookupRangeCheckConfig::configure(
-            meta,
-            advices[9],
-            constants[0],
-            lookup_table,
-            perm.clone(),
-        );
+        // The following columns need to be equality-enabled for their use in sub-configs:
+        //
+        // add::Config and add_incomplete::Config:
+        // - advices[0]: x_p,
+        // - advices[1]: y_p,
+        // - advices[2]: x_qr,
+        // - advices[3]: y_qr,
+        //
+        // mul_fixed::Config:
+        // - advices[4]: window
+        // - advices[5]: u
+        //
+        // mul_fixed::base_field_element::Config:
+        // - [advices[6], advices[7], advices[8]]: canon_advices
+        //
+        // mul::overflow::Config:
+        // - [advices[0], advices[1], advices[2]]: advices
+        //
+        // mul::incomplete::Config
+        // - advices[4]: lambda1
+        // - advices[9]: z
+        //
+        // mul::complete::Config:
+        // - advices[9]: z_complete
+        //
+        // I have not yet figured out where constants[1] is used in an equality constraint
+        // but we get synthesis errors without it.
```

Comment:
`constants[1]` is used in `chip::mul::Config::assign()` to set `z_init` to zero. This will be replaced by the public inputs API.